modules/fscloud/variables.tf

```diff
@@ -0,0 +1,84 @@
+variable "region" {
+  type        = string
+  description = "The IBM Cloud region where the Cloudant instance will be provisioned."
+  default     = "us-south"
+}
+
+variable "plan" {
+  type        = string
+  description = "The plan for the Cloudant instance. Standard or lite."
+  default     = "standard"
+
+  validation {
+    condition     = contains(["standard", "lite"], var.plan)
+    error_message = "Supported plans: standard or lite. Cloudant instance on dedicated host supports standard plan only."
+  }
+}
+
+variable "resource_group_id" {
+  description = "The Id of an existing IBM Cloud resource group where the instance will be grouped."
+  type        = string
+}
+
+variable "instance_name" {
+  description = "The name of the Cloudant instance"
+  type        = string
+}
+
+variable "legacy_credentials" {
+  type        = bool
+  description = "Use both legacy credentials, in addition to IAM credentials for authentication. If set to false, use use only IAM credentials."
+  default     = false # uses only IAM credentials
```

Comment:
hmm, we may need to enforce IAM credentials for fscloud - we should find out

Reply:
We should find this out before we create the DA. Can you ask in our channel perhaps?

Reply:
Checked in the channel. We can go ahead just with IAM for FSCloud module.